releases/net472/KnownIssues/706819 - Winforms applications may have problems rendering UI on latest Operating Systems .md

```diff
@@ -0,0 +1,11 @@
+ # Winforms application may have problems rendering application UI and instead, may render a white screen with a red ‘X’ mark across.
+
+## Symptoms
+Winforms application may have problems rendering application UI and instead, may render a white screen with a red ‘X’ mark across.
+## Cause
+The issue is caused by a latest change in windows OS that exposed a bug in .NET 4.7.2 source code. Latest Windows operating system is now adding a new flag to VisualStyle state when application UI launches a child  dialog (i.e  FileSaveDialog or PrintpreviewDialog etc) that cause .NET framework source to fail on VisualStylestate check and thus throw an exception while rendering the UI.
```

Comment:
Mention the specific OS as this specific to RS5. Same for title header. 
"The issue is caused by a change in Windows 10 October 2018 Update that exposed a bug in .NET Framework 4.7.2."

Reply:
Okay

Reply:
- is now adding --> now adds
- application UI --> the application UI
- i.e --> i.e., a
- delete "etc
- cause .NET framework --> causes .NET Framework
- delete "source"; the source doesn't fail; it's the binary
- fail on VisualStylestate check --> fail its VisualStyleState check